### DISCUSSION TITLE
Starting with a Blank Slate

### DISCUSSION THREAD

**Discussion by sungdzda** [microweber/microweber#899] [General]
Hello,

Today I installed Microweber on my server. During initial setup, it seemed to me that I was forced to pick a template from a selection of templates. I do not mind using templates as a starting point, except that all provided templates have branding in them on the bottom right corner. I wonder if it is possible to either **a)** start with a blank page and build everything myself from scratch, or **b)** remove branding on existing templates. I am not interested in purchasing a white label license, just to be perfectly clear.

Thank you.

**Comment by peter-mw** [comment]
Hi, you can use the template bootstrap5 and remove the footer text 

https://github.com/microweber-templates/bootstrap5
